e2e/cypress/integration/ui/settings/project_mgmt.spec.ts

```diff
@@ -1,31 +1,33 @@
-import { itFlaky } from '../../../support/constants';
-interface Project {
-  id: string;
-  name: string;
-  type: string;
-}
+import { Project, Rule } from '../../../support/types';
 
 describe('project management', () => {
-  // we increase the default delay to mimic the average human's typing speed
-  // only need this for input values upon which later test assertions depend
-  // ref: https://github.com/cypress-io/cypress/issues/534
-  const typeDelay = 50;
+
   let adminIdToken = '';
   const now = Cypress.moment().format('MMDDYYhhmm');
-  const cypressPrefix = 'cypress-test';
+  const cypressPrefix = 'proj-mgmt';
   const projectID = `${cypressPrefix}-project1-${now}`;
-  const customProjectID = `${cypressPrefix}-customproject-${now}`;
   const projectName = `${cypressPrefix} project1 ${now}`;
-  const ruleID = `${cypressPrefix}-rule-${now}`;
-  const ruleName = `${cypressPrefix} rule ${now}`;
+
+  const customWithoutPolsProjectID = `${cypressPrefix}-custom1-${now}`;
+  const customWithPolsProjectID = `${cypressPrefix}-custom2-${now}`;
+  const associatedPolicies = ['project-editors', 'project-viewers', 'project-owners'];
 
   before(() => {
     cy.adminLogin('/settings/projects').then(() => {
       const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
       adminIdToken = admin.id_token;
-      cy.cleanupIAMObjectsByIDPrefixes(cypressPrefix, ['projects', 'policies']);
-      // Set the projects filter to all projects
-      cy.applyProjectsFilter([]);
+      cy.cleanupIAMObjectsByIDPrefixes(cypressPrefix, ['projects']);
+
+      // if there are projects that already exist, set the projects filter to all projects
+      cy.request({
```

Comment:
might want to turn this into a helper, like `clearProjectsFilter`

Reply:
if we're not doing it here, that would be a helpful comment within the code so as we duplicate the work it stands out

Reply:
Looks like this is the only instance we're using it, so I'll leave this as is and updatethe comment